ingest/ingest.go

```diff
@@ -0,0 +1,56 @@
+package ingest
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+const (
+	resourcePath = "/log/ingest"
+	method       = "POST"
+)
+
+type Log struct {
+	Message    string            `json:"msg"`
+	Timestamp  time.Time         `json:"timestamp"`
+	ResourceId map[string]string `json:"_lm.resourceId"`
+}
+
+type Ingest struct {
+	HostUrl   string
+	AccessID  string
+	AccessKey string
+}
+
+func (in *Ingest) SendLogs(logs []Log) (*Response, error) {
```

Comment:
I think the ingest lib should handle batching so it's very easy to use, like https://github.com/unomaly/ingest-go

Reply:
I disagree, I think we should give the client flexibility to batch it or use at however it wants.

https://docs.datadoghq.com/logs/log_collection/?tab=http#application-log-collection

Reply:
I think batching is valuable, but I also think that we can add it later. This API would allow us to add batching without breaking anything. We would maybe add timeout and size to the `Ingest` struct above, and transparently handle batching behind the `SendLogs` function.